docs/resources/generate.md

```diff
@@ -3,12 +3,18 @@
 page_title: "liqo_generate Resource - liqo"
 subcategory: ""
 description: |-
-  Generate peering parameters for remote clusters
+  Generate resource retrieves the information concerning the local
```

Comment:
I would simply say " It retrieves the information...".
If you agree, please propagate this change across the documentation.